react/features/base/tracks/functions.js

```diff
@@ -216,6 +216,25 @@ export function isRemoteTrackMuted(tracks, mediaType, participantId) {
     return !track || track.muted;
 }
 
+/**
+ * Returns whether or not the current environment needs a user interaction with
+ * the page before any unmute can occur.
+ *
+ * @param {Object} state - The redux state.
+ * @returns {boolean}
+ */
+export function isUserInteractionRequiredForUnmute(state) {
+    const { browser } = JitsiMeetJS.util;
```

Comment:
`browser` is exported from lib-jitsi-meet. Isn't it better to just import it?

Reply:
I'd move it to browsercapabilities as you suggested.

Reply:
Actually browser is the exported `BrowserCapabilities` instance from lib-jitsi-meet, isn't it? So I guess we'll still need the reference in order to call the newly created method (that will basically include all the browser checks in it).